### PR TITLE
Inventory Pickup: Toolbox, Wires, Balloons, Fluorescent Tubes, Samples

### PR DESCRIPTION
**Pull request by voidsong-dragonfly**
Adds (almost) direct-to-inventory pickup of the mentioned IE blocks/items.

Items are dropped at the player position without any pickup delay to let items such as the Satchel (from Thermal) process their auto-insert-on-pickup actions. This was recommended by KL on the CoFH discord so I'm gonna follow best practice.

Balloon uses the `getBlock()` from the blockstate to get the balloon itemstack as I could not find a reasonable better way.

All manual entries have been updated as needed.

### CODE REVIEW

File: src/main/resources/assets/immersiveengineering/manual/en_us/balloon.txt [hunk 1]
```diff
@@ -3,6 +3,6 @@ Balloon
 <&recipe>Balloons are sacks of fabric with a heatsource below to keep them afloat. They will emit light, but despite the torch, they are not a fire hazard.
 Balloons can be placed like normal blocks, or when no block is targeted, they will be placed in the air infront of you.
 Similar to <link;metalconstruction;Structural Connectors;connector_recipes> you can attach Hemp Rope and Steel Wire to the balloon.
-Sneak-clicking allows you to set a height offset for the balloon, allowing you to place it up to 5 blocks above the target.
+Sneak-clicking while placing allows you to set a height offset for the balloon, allowing you to place it up to 5 blocks above the target. Sneaking and right-clicking will pick the balloon up.
```
Comment (BluSunrize) [inline src/main/resources/assets/immersiveengineering/manual/en_us/balloon.txt]: I think I'd prefer "Sneaking and right-clicking on a balloon will let you pick it up."

File: src/main/resources/assets/immersiveengineering/manual/en_us/tesla_coil.txt [hunk 2]
```diff
@@ -6,4 +6,4 @@ The coil is disabled unless supplied with a redstone signal. Rightclicking it wi
...
-<&tube_recipes>Fluorescent tubes glow when placed inside a strong high-frequency electric field. Such a frequency is generated by any tesla coil, so placing a tube near an active tesla coil will cause it to glow. The range is about 1.5 times the range in which you will be hit by lightning. They can be colored in the engineer's workbench and rotated by shift + rightclicking them with an Engineer's Hammer.
\ No newline at end of file
+<&tube_recipes>Fluorescent tubes glow when placed inside a strong high-frequency electric field. Such a frequency is generated by any tesla coil, so placing a tube near an active tesla coil will cause it to glow. The range is about 1.5 times the range in which you will be hit by lightning. They can be colored in the engineer's workbench and rotated by shift + rightclicking them with an Engineer's Hammer. Tubes can be picked up directly by shift + rightclicking with an empty hand.
```
Comment (BluSunrize) [inline src/main/resources/assets/immersiveengineering/manual/en_us/tesla_coil.txt]: Both references to "shift" should be replaced with "sneaking" here.

**Review by BluSunrize — CHANGES_REQUESTED** [review]
Minor changes to the manual please.

**Comment by voidsong-dragonfly** [comment]
Changes should be done now, & the balloon entry cleaned up with linebreaks to be a bit more readable.

**Review by BluSunrize — APPROVED** [review]

**Pull request merged**